src/snowflake/cli/api/project/util.py

```diff
@@ -88,6 +88,11 @@ def is_valid_object_name(name: str, max_depth=2, allow_quoted=True) -> bool:
     return re.fullmatch(pattern, name) is not None
 
 
+def quote_identifier(identifier: str) -> str:
+    # double quote the identifier
+    return '"' + identifier.replace('"', '""') + '"'
```

Comment:
technically what's passed in is not an identifier, it's a string. This function is rather dangerous as written, from the calling side we could end up quoting a quoted ID and it would look fine. How about to_quoted_identifier(value: str), and if the value is already quoted, do nothing, otherwise quote it? It would be a bit safer.

Reply:
good point. Updated.